CNAME

```diff
@@ -0,0 +1 @@
+docs.darwinia.network
```

Comment:
How long does it take to deploy the documentation in the Vercel? @fewensa  Would you like to revert the test staff and merge the real CHAME file into main branch first?